src/shared-components/EventForm/EventForm.spec.tsx

```
@@ -0,0 +1,1732 @@
+      return (
+        <div data-testid="customRecurrenceModalMock">
+          <button
+            data-testid="updateRecurrenceRule"
+            onClick={() => {
+              const newRule = createDefaultRecurrenceRule(
+                new Date('2025-01-01'),
+                Frequency.DAILY,
+              );
+              setRecurrenceRuleState(newRule);
+            }}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)